app/controllers/review_queues_controller.rb

```diff
@@ -36,6 +48,24 @@ def submit
     render json: { status: 'ok' }
   end
 
+  def item
+    @item = ReviewItem.find(params[:item_id])
+    render 'queue.html.erb'
+  end
+
+  def reviews
+    @all = params[:all] == '1'
+
+    @reviews =
+      if @all
+        ReviewResult
```

Comment:
Needs a `.all`

Reply:
Where?  Does it still need that if it's paginated?